kentikapi/client_integration_test.go

```diff
@@ -417,3 +425,110 @@ func timePtr(v time.Time) *time.Time {
 func durationPtr(v time.Duration) *time.Duration {
 	return &v
 }
+
+func TestGRPCClient_GetAgent(t *testing.T) {
+	tests := []struct {
+		name    string
+		id      string
+		res     *pb.GetAgentResponse
+		errCode codes.Code
+		errMsg  string
+	}{
+		{
+			"invalid request with wrong id",
```

Comment:
```suggestion
			"request with invalid id, status NotFound received",
```
In fact the server returns code 5 NotFound in such case. It also returns following text: "Could not find synthetic test with id 4417". Please modify the test to reflect that (error code is important). The user should get such error message. For now we can test that error message contains such substring.

Please adjust expectedResponse, expectedErrorCode and expectedErrorMsg accordingly.

Reply:
Done